provider/oci/environ.go

```diff
@@ -649,12 +677,16 @@ func (o *Environ) terminateInstances(instances ...*ociInstance) error {
 			defer wg.Done()
 			if err := inst.deleteInstance(); err != nil {
 				errCh <- instError{id: inst.Id(), err: err}
+				// todo(tsm) add ctx as argument
```

Comment:
remove these comments
you need to do the error processing down below as you iterate the errCh